server/apps/publish/archive_publish_tests.py

```diff
@@ -544,6 +548,12 @@ def test_remove_published_and_killed_content_separately(self):
             published_items = published_service.get_other_published_items(original[config.ID_FIELD])
             self.assertEquals(1, published_items.count())
 
+            article_in_production = get_resource_service(ARCHIVE).find_one(req=None, _id=original[config.ID_FIELD])
+            self.assertIsNotNone(article_in_production)
+            self.assertEquals(article_in_production[config.CONTENT_STATE], 'published')
+            self.assertEquals(article_in_production[config.VERSION], original[config.VERSION] + 1)
```

Comment:
`assertEquals()` has been deprecated in Python3.3, I suggest replacing it with `assertEqual()` throughout the file.